lms/djangoapps/django_comment_client/utils.py

```diff
@@ -137,6 +138,29 @@ def get_accessible_discussion_modules(course, user, include_all=False):  # pylin
     ]
 
 
+def get_discussion_module(course, user, discussion_id):
```

Comment:
1. I was trying to understand what type of code reuse and/or additions might be needed by reviewing `discussion_category_id_access`, but this just got me more confused.  Do you know what a Category is?  Is it the same or different from a Topic?
2. I think we need to enhance our comments or something to better understand how and when a Topic ID can become a Discussion ID.  Are these really always the same thing?  Is there any difference?  For example, if these comments said "... a discussion id (a.k.a topic id)", would that be 100% accurate always?